doc/api/tls.md

```diff
@@ -1027,6 +1027,11 @@ added: v0.3.2
     force SSL version 3. The possible values depend on the version of OpenSSL
     installed in the environment and are defined in the constant
     [SSL_METHODS][].
+  * `secureOptions` {number} The options via bitmask affecting the protocol
```

Comment:
I'm having trouble determining how to specify the default behavior because I'm unsure whether we want to disclose the underlying implementation to the user. The setting itself doesn't really default to anything; however, `SSL_OP_NO_SSLv2` and `SSL_OP_NO_SSLv3` are always set on the SSL context; and `_tls_common.createSecureContext` is adding `SSL_OP_CIPHER_SERVER_PREFERENCE` if the `honorCipherOrder` options is true. Is this something that we wish to describe to the user in this section?